Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Select.hs

```diff
@@ -65,8 +65,9 @@ handler merchant sReq estimate = do
   searchReq <- QSR.findById estimate.requestId >>= fromMaybeM (SearchRequestNotFound estimate.requestId.getId)
   QDQ.setInactiveAllDQByEstId sReq.estimateId now
   farePolicy <- getFarePolicy searchReq.merchantOperatingCityId estimate.vehicleVariant searchReq.area
+  let customerCancellationDues = listToMaybe [eb.price.value | eb <- estimate.estimateBreakupList, eb.title == "CUSTOMER_CANCELLATION_DUES"]
```

Comment:
Didn't get the logic here? Why you are fetching it from estimate table, when you have the same value inside searchReq table.

Reply:
We are not storing in searchRequest table